src/modules/modals/TransferTable.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+	<NcModal v-if="showModal"
+		size="normal"
+		@close="actionCancel">
+		<div class="modal__content" data-cy="transferTableModal">
+			<div class="row">
+				<div class="col-4">
+					<h2>{{ t('tables', 'Transfer table') }}</h2>
+				</div>
+			</div>
+			<div class="row">
+				<div class="col-4 mandatory space-T">
+					{{ t('tables', 'Owner') }}
+				</div>
+
+				<!-- We don't need to show this, since the current user will only see this if they're the owner? -->
+				<div v-if="localTable !== undefined" class="col-3 inline space-T-small">
+					<NcUserBubble
+						:margin="4"
+						:size="30"
+						:display-name="localTable.ownerDisplayName"
+						:user="localTable.owner" />
+				</div>
+			</div>
+			<div class="row">
+				<NcUserAndGroupPicker :select-users="true" :select-groups="false" :new-owner-user-id.sync="newOwnerUserId" />
+			</div>
+			<div class="row">
+				<div class="fix-col-4 space-T justify-between">
+					<NcButton type="warning" :disabled="newOwnerUserId === ''" data-cy="transferTableButton" @click="transferMe">
```

Comment:
Maybe we can right align the button as the primary action is usually on the right.